client/src/main/java/com/battleships/gamewindow/board/PlayerBoard.java

```diff
@@ -55,4 +55,27 @@ public BoardField getField(Coordinate coordinate) {
         return board.get(coordinate);
     }
 
+
+    public boolean verifyShot(Shot shot) {
+        return fleet.isHit(Coordinate.fromShot(shot));
+    }
+
+    public List<BoardField> getHitMastsCoordinates(Shot shot) {
+        List<BoardField> hitResult = fleet.returnBoardFieldsAfterShot(Coordinate.fromShot(shot));
+        List<BoardField> buffer = new ArrayList<>();
+        for (BoardField boardField : hitResult) {
+            if (boardField.isSunk()) {
+                buffer.addAll(
+                        bufforCalculator.calculateBuffer(
+                                fleet.getShipForCoordinate(
+                                        boardField.getCoordinate())));
+            }
```

Comment:
Level of indentation is fu#$ed up. It's really unreadable. Wrap it, to method, to class idk

Reply:
fixed